frontend/src/component/common/ToastRenderer/Toast/Toast.styles.ts

```diff
@@ -2,56 +2,42 @@ import { makeStyles } from 'tss-react/mui';
 
 export const useStyles = makeStyles()((theme) => ({
     container: {
-        maxWidth: '450px',
-        background: theme.palette.background.paper,
+        alignItems: 'center',
+        background:
+            theme.mode === 'light' ? '#201E42' : theme.palette.background.paper,
+        borderRadius: theme.shape.borderRadiusMedium,
         boxShadow: theme.boxShadows.popup,
-        zIndex: 500,
+        display: 'flex',
+        flexDirection: 'row',
+        gap: theme.spacing(2),
         margin: '0 0.8rem',
-        borderRadius: '12.5px',
-        padding: '2rem',
-    },
-    innerContainer: {
-        position: 'relative',
+        maxWidth: '450px',
+        padding: theme.spacing(1),
+        paddingInlineStart: theme.spacing(2),
+        zIndex: 500,
```

Comment:
As we touch this code shouldn't we switch zIndex to something from a design system token?